src/Socket.php

```diff
@@ -824,6 +829,13 @@ public static function showForLocation($item)
         $number = countElementsInTable('glpi_sockets', ['locations_id' => $ID ]);
 
         if ($canedit) {
+            echo '<table class="tab_cadre_fixe">';
+            echo '<tbody>';
+            echo '<tr class="tab_bg_1">';
+            echo '<th colspan="2" class="">';
+            echo __('Add socket');
+            echo '</th></tr></tbody><table>';
+
```

Comment:
I guess you made a mistake here. You added new features to a deprecated method.

Reply:
fixed